pkg/remote-config/types.go

```diff
@@ -38,10 +38,11 @@ type RemoteConfig struct {
 }
 
 type RemoteConfigResponse struct {
-	Id        string     `json:"id"`
-	Domain    string     `json:"domain"`
-	Name      string     `json:"name"`
-	Endpoints []Endpoint `json:"endpoints"`
+	Id           string     `json:"id"`
+	Domain       string     `json:"domain"`
+	Name         string     `json:"name"`
+	ProxyRequest bool       `json:"proxyRequest"`
```

Comment:
note: this is at the vendor level, but gets set at the individual endpoint in the cache. This might require I re-think how the cache is formatted since it could be the case that we havent detected the endpoint but still want to proxy the request for the vendor.

Reply:
actually - since we arent supporting the concept of "create vendor" where api keys would live, and instead have a separate "keys" section - this probably will have to come in as a separate config.

for each key we'll need to map to an fqdn. We might find that a credential has multiple valid fqdns.